src/test/java/net/imglib2/algorithm/neighborhood/RandomAccessiblePairWithNeighborhoodTest.java

```diff
@@ -0,0 +1,82 @@
+ * %%
+ * Copyright (C) 2009 - 2016 Tobias Pietzsch, Stephan Preibisch, Stephan Saalfeld,
+ * John Bogovic, Albert Cardona, Barry DeZonia, Christian Dietz, Jan Funke,
+ * Aivar Grislis, Jonathan Hale, Grant Harris, Stefan Helfrich, Mark Hiner,
+ * Martin Horn, Steffen Jaensch, Lee Kamentsky, Larry Lindsey, Melissa Linkert,
+ * Mark Longair, Brian Northan, Nick Perry, Curtis Rueden, Johannes Schindelin,
+ * Jean-Yves Tinevez and Michael Zinsmaier.
+ * %%
+ * Redistribution and use in source and binary forms, with or without
+ * modification, are permitted provided that the following conditions are met:
+ * 
+ * 1. Redistributions of source code must retain the above copyright notice,
+ *    this list of conditions and the following disclaimer.
+ * 2. Redistributions in binary form must reproduce the above copyright notice,
+ *    this list of conditions and the following disclaimer in the documentation
+ *    and/or other materials provided with the distribution.
+ * 
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+ * AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+ * IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE
+ * ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDERS OR CONTRIBUTORS BE
+ * LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR
+ * CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF
+ * SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS
+ * INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN
+ * CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
+ * ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+ * POSSIBILITY OF SUCH DAMAGE.
+ * #L%
+ */
+package net.imglib2.algorithm.neighborhood;
+
+import net.imglib2.Cursor;
+import net.imglib2.FinalDimensions;
+import net.imglib2.RandomAccess;
+import net.imglib2.RandomAccessible;
+import net.imglib2.RandomAccessibleInterval;
+import net.imglib2.img.Img;
+import net.imglib2.img.array.ArrayImgFactory;
+import net.imglib2.outofbounds.OutOfBoundsBorderFactory;
+import net.imglib2.outofbounds.OutOfBoundsFactory;
+import net.imglib2.type.numeric.integer.ByteType;
+import net.imglib2.util.Pair;
+import net.imglib2.view.Views;
+
+import org.junit.Test;
+
+public class RandomAccessiblePairWithNeighborhoodTest
+{
+
+	@Test
+	public void testRandomAccessOnPair()
+	{
+		// Extend input with Views.extend()
+		Img< ByteType > in = new ArrayImgFactory< ByteType >().create( new FinalDimensions( 3, 3 ), new ByteType() );
+		OutOfBoundsFactory< ByteType, RandomAccessibleInterval< ByteType > > outOfBoundsFactory =
+				new OutOfBoundsBorderFactory<>();
+		RandomAccessibleInterval< ByteType > extendedIn = Views.interval( Views.extend( in, outOfBoundsFactory ), in );
+
+		// Create RandomAccessiblePair
+		final RandomAccessible< Neighborhood< ByteType > > safe = new RectangleShape( 1,
```

Comment:
` 1,` should be in a new line like this (with appropriate indentation):
```java
... = new RectangleShape( 
				1,
				false )
```

Reply:
@hanslovsky I have added an additional comment in the opening line so that the formatter doesn't try to move the `1` to the first line every time it's run. Let me know if I should remove it or keep it that way...

Reply:
That's interesting. Are you using eclipse? I have noticed that the eclipse formatter is not always consistent among different machines (probably dependent on what settings are being used). I think the comment is fine here.

Reply:
I was using Eclipse Neon on macOS with a weird configuration (that I don't have a clue where I got it from) instead of the `imglib2-eclipse-mars-clean-up-formatter-style.xml`. Sorry for the confusion!